SignalRServiceBenchmarkPlugin/framework/master/ArgsOption.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System.Collections.Generic;
+
+namespace Rpc.Master
+{
+    public class ArgsOption
+    {
+        // Rpc 
+        [Option("RpcPort", Required = false, Default = 5050, HelpText = "")]
```

Comment:
Please add HelpText for all fields, especially those have default values. You'd better tell user the default value in help description.

Reply:
Added